base/main_test_cluster.go

```diff
@@ -202,7 +202,11 @@ func initV2Cluster(server string) *gocb.Cluster {
 }
 
 func (c *tbpClusterV2) getBucketNames() ([]string, error) {
-	manager := c.cluster.Buckets()
+
+	cluster := getCluster(c.server)
+	defer cluster.Close(nil)
```

Comment:
I was thinking of a `closeCluster` helper here (and in the other defer cases below) to keep errcheck happy without the awkward inline function pattern.

```go
func (c *tbpClusterV2) closeCluster(cluster *gocb.Cluster) {
	if err := cluster.Close(nil); err != nil {
		c.logger(context.Background(), "Couldn't close cluster connection: %v", err)
	}
}
```

```suggestion
	defer c.closeCluster(cluster)
```